### ISSUE TITLE
Can you add the option of ly display manager?

### ISSUE THREAD

**Issue by RobotsAreCrazy** [ChrisTitusTech/Debian-titus#77]
Hi Chris, great fan of your work

Wondering if you could add support in your script for ly display manager please as I wanted to use it instead of sddm, but unsure what to do as in the script it mentions sddm themes....

Thankyou

**Comment by nxjosephofficial** [comment]
> Also, would this also require an install without an desktop environment?
No, this doesn't matter. You can select your preferred environment in the display manager.

I'm not sure but it should work without any modify.

**Comment by nxjosephofficial** [comment]
Ah, yes. Brave's arch is amd64 in the script. Let me see if brave has arm64 build.

**Comment by RobotsAreCrazy** [comment]
Tbh if I can't use brave, maybe flatpak as can then use librewolf

**Comment by nxjosephofficial** [comment]
Actually it is easy to modify this script. You just put commands that you need to install packages.
